README.md

```diff
@@ -58,43 +58,73 @@ test and production environments.
 
 # Using a Pocket Android Plugin
 Just import the `PocketAion` class and call any of the functions described below.
+```
+import network.pokt.aion.PocketAion;
+```
 
 ## The Configuration object
 The constructor for any given `PocketAion` instance requires a class implementation of the `Configuration` interface, 
 and an instance of your Android application `Context`. Let's take a look at the example below:
 
 ```
-PocketAion pocketAion = new PocketAion(new Configuration() {
+import network.pokt.pocketsdk.interfaces.Configuration;
+
+public class testConfig implements Configuration {
     @Override
     public URL getNodeUrl() throws MalformedURLException {
         return new URL("https://aion.pokt.network");
     }
-}, appContext);
+}
+```
+After setting our `Configuration class`, we will need to initialize a new instance of the `PocketAion class`:
+```
+try {
+    PocketAion pocketAion = new PocketAion(new PocketAionConfiguration(), context);
+} catch (InvalidConfigurationException e) {
+    e.printStackTrace();
+}
 ```
 
 ## Creating and Importing a Wallet
-Follow the following example to create an AION Wallet:
+The two requirement you need to specify for creating a wallet is:
+
+1- Subnetwork
+
+2- Data
+
+Once those have been defined, enter: 
 
 ```
-Wallet newWallet = pocketAion.createWallet("32", null);
+try {
+    Wallet wallet = this.pocketAion.createWallet("32", null);
+} catch (CreateWalletException cwe) {
+    throw cwe;
+}
 ```
 
-And to import:
+And to import, you need to define:
+	
+1- Public Key
 
-```
-String privateKey = "0x";
-String address = "0x";
+2- Private Key
+	
+3- Subnetwork
+
+4- Data
 
-Wallet importedWallet = pocketAion.importWallet(privateKey, "32", address, null);
+```
+try {
+    Wallet wallet = this.pocketAion.importWallet("0x0...", "32", "0x0...", null);
+} catch (ImportWalletException iwe) {
+    throw iwe;
+}
 ```
 
 ## Querying Data
-Currently there are 2 supported namespaces in Pocket Node for AION: `net` and `eth`.
-In the examples below, you will see how to query the supported RPC calls in both 
-namespaces.
+Currently there are 2 supported namespaces in PocketAION: `net` and `eth` that contains prebuild `RPC_Methods` to call when executing each namespace. In the examples below, you will see how we get the balance of a wallet using both namespaces.
```

Comment:
```net``` doesn't let you get balance from the blockchain, only the ```eth``` namespace have the getBalance. replace with ```In the examples below, you will see how you can use both namespaces.```